contracts/Core/AssetManager.sol

```diff
@@ -78,7 +93,10 @@ contract AssetManager is ACL, AssetStorage {
         string calldata selector, 
         string calldata name, 
         bool repeat
-    ) external payable {
+    ) 
+        external 
+        onlyRole(parameters.getAssetCreatorHash()) 
```

Comment:
By Admin Access to Jobs, do we mean that only admin can create jobs ?

Reply:
Yes, only admin can create a job/collection